stock_quote.gemspec

```diff
@@ -25,7 +25,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'webmock', '~> 1.24.6'
   s.add_development_dependency 'rubocop', '~> 0.40'
   s.add_development_dependency 'byebug', '~> 9.0.5'
-  s.add_runtime_dependency 'rest-client', '~> 1.8'
+  s.add_development_dependency 'pry', '   ~> 0.10.4'
```

Comment:
I understand, here's the change https://github.com/tyrauber/stock_quote/pull/37/commits/e9c04865d0b8b7593400d75d858fe34b950d9c54